core/src/executor/execute.rs

```diff
@@ -36,6 +36,9 @@ pub enum ExecuteError {
 
 #[derive(Serialize, Debug, PartialEq)]
 pub enum Payload {
+    ShowColumns {
+        rows: Vec<Vec<String>>,
+    },
```

Comment:
`Vec<Vec<String>>` representation covers more than what we actually want to describe.
Rather than using superset, let's use identical data structure for this.
```rust
// use ast::DataType

ShowColumns {
    columns: Vec<(String, DataType)>,
}
```

Then, `ast::DataType` is explicitly exposed to the user.
Let's add `ast::DataType` to `prelude`.
We don't need to use `prelude` in `core`, but it would be good to use in `test-suite` which you already used well.

In `/core/src/lib.rs`,
<img width="483" alt="image" src="https://user-images.githubusercontent.com/2025065/163664260-80aedf68-73af-4c2e-9a22-ad6c5349abe6.png">